resources/careconnect-to-ukcore/allergyintolerance/input/AllergyIntoleranceDescriptionId-Extension-3to4_001.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Filename should be AllergyIntoleranceExtension-coding-sctdescid-3to4_001.json (001 as there is a 000 file already.)

This needs to match the map name, i.e. AllergyIntoleranceExtension-coding-sctdescid-3to4.map